app/app/src/main/AndroidManifest.xml

```diff
@@ -7,6 +7,10 @@
     <uses-permission android:name="android.permission.WRITE_SETTINGS" />
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
 
+    <queries>
+        <package android:name="io.shoonya.shoonyadpc" />
+    </queries>
+
     <application
```

Comment:
Lets add a comment here if possible in Manifest file that this is needed for Android 11 and above devices.

Reply:
Sure... it will bring more clarity on why we're doing 👍